public/index.html

```diff
@@ -209,9 +230,9 @@ <h3>How can I contact you or get involved?</h3>
     <a href="/" class="home-link"><img src="assets/images/POAP.svg" alt="" /></a>
     <div class="ecosystem">
       <h4>POAP Ecosystem</h4>
-      <a href="https://www.poap.xyz/" class="a">poap.xyz</a>
+      <!-- <a href="https://www.poap.xyz/" class="a">poap.xyz</a> -->
       <a href="https://poap.fun/" class="b">poap.fun</a>
-      <a href="https://poap.chat/" class="c">poap.chat</a>
+      <!-- <a href="https://poap.chat/" class="c">poap.chat</a> -->
```

Comment:
Why are we removing these links? If they should be removed, lets balance the ecosystem links (right now because we are using a grid display it just leaves an empty space)
![image](https://user-images.githubusercontent.com/19996944/148089018-6c780995-54f3-4fab-a5b3-6e7178b0369e.png)

Reply:
They were requests from Isabel: 

- Links now aligned
- The poap.xyz is because it it the page itself
- The poap.chat is deprecated